05_Customizing_OrbitDB/01_Definitions.md

```diff
@@ -0,0 +1,58 @@
+# What are Stores, AccessControllers and Indicies.
+
+Before we start to implement
+our custom store, access controller and index
+to allow our users to comment on each others
+notes, we should probably define what a store,
+AccessController and Index are and what
+function they have in OrbitDB.
+
+## The Store
+You have already worked with several
+Stores in the Tutorial.
+The `docstore` was used to store
+the individual pieces (or rather their CIDs)
+and the `kvstore` type was used to
+represent a user.
+
+Both of those were instances of a Store.
+They stored some data and provided easy
+functions to access them, like `get`, `put` or `set`,
+with which you could easily interact with
+the database itself.
+
+## The Index
+But a store doesn't actually
+store the current state of the database,
```

Comment:
We should somehow make clear that it's storing it in RAM

Reply:
Yes. That's important.